utils/scripts/ocm/ocm.py

```diff
@@ -1179,6 +1179,7 @@ def change_cluster_channel_group(self):
         ret = execute_command(run_change_channel_cmd)
         if ret is None:
             log.info("Failed to update the channel to {}".format(self.cluster_name))
+            sys.exit(1)
```

Comment:
is this funciton going to be used in Robot Framework? maybe tasks. If yes, then we should aboid sys.exit. You could check the solution applied in functions like install_rhoam_addon